### ISSUE TITLE
Dirt Rally doesn't work

### ISSUE THREAD

**Issue by 1wsx10** [LibreVR/Revive#869]
i get this when i try to start the game: https://imgur.com/a/XIVkD

i have had the game running before, some time early 2016.

i have reinstalled the game

i have reinstalled revive and oculus home

i have tried the instructions for dirt here: https://github.com/LibreVR/Revive/wiki/Compatibility-list

i have tried ghost's fix here: https://github.com/LibreVR/Revive/issues/283
this one just causes it to crash rather than hang

i have tried running the game (drt.exe) in compatability mode for windows xp SP3, 7, 8, setting it to run as admin, setting it to 'disable fullscreen optimizations'

**Comment by 1wsx10** [comment]
i'm on windows 10 home

using GTX 1070, driver: 388.31
i just checked, there is an update. ill install that and get back

**Comment by 1wsx10** [comment]
that didn't fix it

**Comment by Bashinbagels** [comment]
I am also having the same problem. Can't seem to figure it out.

**Comment by jb600cbr** [comment]
same problem for me....

**Comment by lewisscrivens98** [comment]
Same here...

**Comment by jb600cbr** [comment]
downgrade to the last version of revive and it's work fine for me.

**Comment by Kosta73** [comment]
Hello,
Me and friend tried to get this work now several hours. Tried different versions of Revive and always patched drt.exe with new Revive version. None of the Revive version wont work.
It will give Dirt Rally logo on your desktop and from vr glasses (int his case Samsung phones) i can see it tries to load something and then just stops.
So we are using combo RiftCat + Vridge for GearVR + PC also tried with normal Vridge for cardboard version. So far nothing has helped.
Those who have it working, do you have Nvidia or Ati card? Do you use wifi or USB connection with your phone?
I have Ivry for iPhone and it works like a charm, but we need to get this work on android phones too as my friend does not have  iPhone.

**Comment by syberphunk** [comment]
Installed Dirt Rally, Oculus software and Revive. Latest version(s) don't work. Also tried patching drt.exe, doesn't work.

Windows 10 64bit Pro, GeForce 1080.

**Comment by Bashinbagels** [comment]
Reverting to revive version 1.1.11 did the trick for me!

**Comment by 1wsx10** [comment]
ok so its the latest version, not just me. hopefully the devs fix it for the next version!

**Comment by Armada651** [comment]
I've tried that latest version with the SteamVR version of Dirt Rally, I can't reproduce the issue it works fine for me.

If anyone wants to do more in-depth troubleshooting add me on discord: Armada#2798

**Comment by syberphunk** [comment]
Yes I think this is worth troubleshooting further, because now I feel we're being a bit too vague as to what the latest version is, and as the developer you may have something setup differently to us as users. I'm not sure what the best way to converse with you directly on discord is, is it a matter of adding you, or is there a server that's setup for supporting this project in particular?

**Comment by Armada651** [comment]
Just add me as a friend.
